build.gradle.kts

```diff
@@ -113,6 +116,8 @@ dependencies {
     api("sc.fiji:bigdataviewer-core:10.4.14")
     api("sc.fiji:bigdataviewer-vistools:1.0.0-beta-28")
     api("sc.fiji:bigvolumeviewer:0.3.3") {
+        exclude("sc.fiji", "bigdataviewer-core")
```

Comment:
I would suggest to add a comment explaining what it accomplishes to exclude bigdataviewer-core and bigdataviewer-vistools here, when they are included as direct dependencies immediately above.